corehq/tests/noseplugins/dividedwerun.py

```diff
@@ -84,7 +84,7 @@ def skip():
                         skip.__name__ = "*"
                     skip.__module__ = name
             else:
-                desc = test.test
+                desc = test.test.descriptor
```

Comment:
I'm not convinced - nose says this:
```
* descriptor -- the function, other than the test, that should be used
  to construct the test name. This is to support generator functions.
```
So it says that it shouldn't be the test itself, and it seems logical to just pass through the test descriptor, since this is the "descriptor" param.